File: Common/Models/SensorModel.cs
namespace Common.Models
{
    public class SensorModel
    {
        public int Id { get; set; }

        public int PositionIndex { get; set; }

        public int DecimalPlaces { get; set; }

        public string MeasuringUnitName { get; set; } = string.Empty;

        public ModuleModel? Module { get; set; } = null;
    }
}

File: SPA/ClientApp/src/app/models/SensorModel.ts
import { ModuleModel } from "./ModuleModel";

export class SensorModel {
id!: number;
positionIndex!: number;
decimalPlaces!: number;
measuringUnitName!: string;
module: ModuleModel | null = null;

isNew: boolean = false;

    public static ToString(model: SensorModel): string {
        if (model === undefined || model.measuringUnitName === undefined)
            return '';
        return `${model.measuringUnitName} #${model.positionIndex}`
    }
}